### DISCUSSION TITLE
nodemcu random relay power after 1 week

### DISCUSSION THREAD

**Discussion by mtraxiot** [arendst/Tasmota#12969] [Q&A] [answered]
I have used a ESP12E for my Garage door sensor and relay door switch, I was using this same device (albeit different GPIO) for about two years with another firmware.
its been running fine for all of the week when Sunday morning I see both my Garage doors open !!.
I couldn't see any MQTT command or OPENHAB command so I can only conclude its someone on Tasmota or the pins I'm using which fired and opened the doors.

is there any way to diagnose this as its happened twice over the last two or so weeks.

pins table

D0 | GPIO16 | One-wire
-- | -- | --
D1 | GPIO5 | Relay1 Hotwater
D2 | GPIO4 | Relay2 Door
D5 | GPIO14 | Relay3 Door
D7 | GPIO13 | Switch2
RX | GPIO3 | Switch1
UV | 5V | Relay VCC

**Comment by barbudor** [comment, marked as answer]
If it's software, then you should see logs in the console or Mqtt status been published
If not, it's most probably electrical

Everything tasmota does would leave a track...

**Reply by mtraxiot** [reply]
yeah I suspect it might be electrical which will be a hard one to figure out.

**Reply by sfromis** [reply]
What I'm doing is to log MQTT messages, thus being able to see when something happened back in time. Of course, this does not provide insight into what may have happened electrically, like some sort of noise or fluctuation.

**Comment by mtraxiot** [comment]
I guess it will just need to mess around with GPIO pins right?

**Reply by sfromis** [reply]
Hard to tell, without understanding what you mean by that sentence.
